packages/sui-critical-css/src/extract-from-app.js

```diff
@@ -65,12 +100,9 @@ export async function extractCSSFromApp({routes, config = {}}) {
 
     manifest[pathKey] = cssFileName
 
-    const css = await extractCSSFromUrl({
+    const css = await extratCriticalCSSFromUrlRetrier(pathOptions, {
```

Comment:
IMHO it could have a generic name like `extractCSS` because of `extratCriticalCSSFromUrlRetrier` is coupled to implementation.

Also, what do you think about pass all the parameters as a named parameter?
```js
extratCriticalCSSFromUrlRetrier({url, configForMobileDevice, requiredClassNames, retries})
```